apps/builder/app/builder/features/style-panel/sections/advanced/advanced.tsx

```diff
@@ -328,6 +335,64 @@ const $advancedProperties = computed(
   }
 );
 
+const AdvancedProperty = memo(
+  ({
+    property,
+    autoFocus,
+  }: {
+    property: StyleProperty;
+    autoFocus: boolean;
+  }) => {
+    const ref = useRef<HTMLDivElement>(null);
+    const [isVisible, setIsVisible] = useState(false);
+
+    useEffect(() => {
+      if (ref.current == null) {
+        return;
+      }
+
+      const controller = new AbortController();
+
+      ref.current.addEventListener(
+        "contentvisibilityautostatechange",
+        (event) => {
+          // @ts-expect-error event.skipped is not known
+          setIsVisible(!event.skipped);
+        },
+        {
+          signal: controller.signal,
+        }
+      );
+
+      return () => {
+        controller.abort();
+      };
+    }, []);
+
+    return (
+      <Flex
+        ref={ref}
+        css={{
+          contentVisibility: "auto",
+          containIntrinsicSize: "auto 400px",
```

Comment:
so we try to tell it that our content is only 400px even if its more so that it doesn't try to calculate layout for more than that? or whats the idea, what happens if user has a bigger window and more properties and can see more than 400?